docs/specviz/displaying.rst

```diff
@@ -28,7 +28,9 @@ using the Python API.  Both are detailed further below.
 Selecting/Showing Data Sets
 ===========================
 
-Data can be selected and de-selected under the "Data" tab. Here, you can click a checkbox next to the listed data to make the data visible (checked) or invisible (unchecked).
+Data can be selected and de-selected in each viewer's data menu, opened by clicking the 
+|icon-viewer-data-select| button in the top left of the viewer. Here, you can click a 
+checkbox next to the listed data to make the data visible (checked) or invisible (unchecked).
```

Comment:
I feel like this is more accurate, as we also have the 👁️ that toggles visibility. 🤷 

```suggestion
checkbox next to the listed data to make the data loaded (checked) or unloaded (unchecked).
```

Reply:
Okay, maybe "loaded" is also misleading because it is loaded in the app but just not the viewer... My head hurts.